### DISCUSSION TITLE
You love sushi? Tell me your favorite sushi !!

### DISCUSSION THREAD

**Discussion by RyukiYoshida** [NAIST-SE/AttractingContributors#417] [General]
I know you love sushi.
Tell me your favorite sushi!

好きな寿司ネタを教えてください！
邪道ネタ大歓迎

My fork : https://github.com/RyukiYoshida/AttractingContributors-sushi.git

**Comment by aru1702** [comment]
As a foreigner, I do love sushi, either when I eat back in my hometown or here!
I prefer sushi with shrimp, squid, and roll sushi, but I don't know the name.

See my repo: https://github.com/aru1702/AttractingContributors-TouhouFumo
Hop into my discussion: https://github.com/NAIST-SE/AttractingContributors/discussions/407

**Comment by ploychanok** [comment]
I created a pull request to your repo.
Would you mind help mine too? https://github.com/ploychanok/AttractingContributors-WithinThreeWords

**Comment by tmymzkr** [comment]
I done pull request 
please help me
https://github.com/m-tomoya/AttractingContributors-movie
